bitnami/tomcat/Chart.yaml

```
@@ -26,4 +26,4 @@ name: tomcat
 sources:
   - https://github.com/bitnami/bitnami-docker-tomcat
   - http://tomcat.apache.org
-version: 9.4.0
+version: 9.4.1
```

```suggestion
version: 9.5.0
```
Please bump the minor version since you are adding new parameters